### PR TITLE
Update ng13

### PR DESCRIPTION
**Pull request by jdroenner**
i ran

 npx @angular/cli@13 update @angular/core@13 @angular/cli@13 --force 
 npx @angular/cli@13 update @angular/cdk@13 @angular/material@13 --force 

and updated  flex-layout and angular-eslint

There was one small issue:
the  clolor-input-component requires view-enapsulation which is deprecated (and i guess disabled by default). i added it in 4eaf959b81d9eba35214daadb65164635ad8faac and used the ::ng-deep modifier which is recommended to use until this feature is removed.

### CODE REVIEW

**Comment by jdroenner** [comment]
[Angular auf 13 upgraden](https://trello.com/c/feqAS6CL/476-angular-auf-13-upgraden)

File: projects/wave-core-new/src/lib/colors/color-attribute-input/color-attribute-input.component.scss [hunk 1]
```diff
@@ -2,7 +2,7 @@
     font-size: 0.8rem;
 }
 
-mat-form-field >>> .color_cell {
+mat-form-field ::ng-deep .color_cell {
```
Comment (ChristianBeilschmidt) [inline projects/wave-core-new/src/lib/colors/color-attribute-input/color-attribute-input.component.scss:5]: You missed another one:

```
⠼ Compiling with Angular sources in Ivy full compilation mode.WARNING: ▲ [WARNING] Unexpected ">"

    /home/beilschmidt/git/geoengine-ui/projects/wave-core-new/src/lib/colors/color-breakpoint-input/color-breakpoint-input.component.scss:5:17:
      5 │ mat-form-field > > > .color_cell {
        ╵                  ^

```

File: projects/gfbio-app/src/app-theme.scss [hunk 2]
```diff
@@ -1,4 +1,4 @@
-@use '~@angular/material' as mat;
+@use '@angular/material' as mat;
```
Comment (ChristianBeilschmidt) [inline projects/gfbio-app/src/app-theme.scss:2]: In the GFBio-App on Startup:

```
ERROR Error: NG0100: ExpressionChangedAfterItHasBeenCheckedError: Expression has changed after it was checked. Previous value: '1238'. Current value: '1174'.. Find more at https://angular.io/errors/NG0100
    Angular 5
    AppComponent_Template app.component.html:44
    Angular 37
    RxJS 25
    ZoneAwarePromise Angular
    toPromise RxJS
    load config.service.ts:244
    load app-config.service.ts:34
    useFactory app.module.ts:38
    Angular 24
    71736 main.ts:12
    Webpack 7
core.mjs:6449
    Angular 33
    RxJS 25
    ZoneAwarePromise Angular
    toPromise RxJS
    load config.service.ts:244
    load app-config.service.ts:34
    useFactory app.module.ts:38
    Angular 24
    71736 main.ts:12
```

File: projects/wave-core-new/ng-package.json [hunk 3]
```diff
@@ -2,44 +2,7 @@
...
-            "ol/source/TileWMS": "OlTileWmsSource",
-            "ol/source/Vector": "OlSourceVector",
-            "ol/source/XYZ": "XYZ",
-            "ol/style": "style",
-            "ol/style/Fill": "OlStyleFill",
-            "ol/style/Stroke": "OlStyleStroke",
-            "ol/style/Style": "OlStyleStyle",
-            "proj4": "proj4",
-            "vega-embed": "vegaEmbed"
-        }
```
Comment (ChristianBeilschmidt) [inline projects/wave-core-new/ng-package.json:42]: Just for curiosity:
Why could you remove all of this and why are there no more UMD warnings about this?

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

File: projects/wave-core-new/src/lib/colors/color-attribute-input/color-attribute-input.component.scss (hunk 1 above)
Comment (jdroenner) [reply projects/wave-core-new/src/lib/colors/color-attribute-input/color-attribute-input.component.scss:5]: yes i do not know how to solve this. If you remove the ::ng-deep the color input will  miss the white border on text. But there is no suggestion what would fix this. The entire /deep/ >>> ::ng-deep api  is deprecated anyway and will be removed in the future.

File: projects/wave-core-new/ng-package.json (hunk 3 above)
Comment (jdroenner) [reply projects/wave-core-new/ng-package.json:42]: the update script did that. I read somewhere that it is no longer required

Comment (jdroenner) [reply projects/wave-core-new/ng-package.json:42]: fbd735e

File: projects/gfbio-app/src/app-theme.scss (hunk 2 above)
Comment (jdroenner) [reply projects/gfbio-app/src/app-theme.scss:2]: fbd735e

File: projects/wave-core-new/src/lib/colors/color-attribute-input/color-attribute-input.component.scss (hunk 1 above)
Comment (jdroenner) [reply projects/wave-core-new/src/lib/colors/color-attribute-input/color-attribute-input.component.scss:5]: fbd735e

**Review by ChristianBeilschmidt — APPROVED** [review]

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Pull request merged**